# **Tourism Data on Java**

#**Import Library**

In [ ]:
import pandas as pd

#**Memuat Dataset**

In [ ]:
file = "/content/drive/MyDrive/AVD/Java.xlsx"

df = pd.read_excel(file)

df

Unnamed: 0                                  place_name  \
0              0                    Alas Purwo National Park   
1              1                                   Green Bay   
2              2                          De Djawatan Forest   
3              3                             Banyuwangi Park   
4              4                 Tempat wisata di banyuwangi   
...          ...                                         ...   
1012          15                          Telaga Biru Cisoka   
1013          16                     Danau Barat Alam Sutera   
1014          17                             Koja Cliff Park   
1015          18      Citra Raya World Of Wonders Theme Park   
1016          19  Kampung Gempol - Wisata Edukasi & Rekreasi   

                                           reviews_link  rating  \
0     https://serpapi.com/search.json?data_id=0x2dd3...     4.5   
1     https://serpapi.com/search.json?data_id=0x2dd4...     4.7   
2     https://serpapi.com/search.json?data_id=0x2dd3...     4.6   
3     https://serpapi.com/search.json?data_id=0x2dd1...     4.5   
4     https://serpapi.com/search.json?data_id=0x2dd1...     4.2   
...                                                 ...     ...   
1012  https://serpapi.com/search.json?data_id=0x2e42...     4.0   
1013  https://serpapi.com/search.json?data_id=0x2e69...     4.5   
1014  https://serpapi.com/search.json?data_id=0x2e42...     4.0   
1015  https://serpapi.com/search.json?data_id=0x2e42...     4.4   
1016  https://serpapi.com/search.json?data_id=0x2e69...     4.6   

      total_reviews                                            address  \
0              1449  Kutorejo, Kalipait, Tegaldlimo, Banyuwangi Reg...   
1              2322  Dusun Krajan, Sarongan, Pesanggaran, Banyuwang...   
2              6906  Purwosari, Benculuk, Kec. Cluring, Kabupaten B...   
3              1242  Dusun Krajan, Dadapan, Kec. Kabat, Kabupaten B...   
4                37  perum adimas diponegoro No.02, Lingkungan Cuki...   
...             ...                                                ...   
1012           6286  PCCJ+Q6X, Jalan Cigaru, Cisoka, Kec. Cisoka, K...   
1013            153  Jl. Kp. Kosong No.11, RT.001/RW.004, Panunggan...   
1014           1690  M9PX+55H, Cireundeu, Cikuya, Kec. Solear, Kabu...   
1015           4740  Citra Raya Cikupa Jl. Ecopolis Boulevard Utama...   
1016            215  Gg. H. Marjuki Jl. Kunciran Induk/Raya Gempol,...   

            city    province  
0     banyuwangi  jawa timur  
1     banyuwangi  jawa timur  
2     banyuwangi  jawa timur  
3     banyuwangi  jawa timur  
4     banyuwangi  jawa timur  
...          ...         ...  
1012    tangeran      banten  
1013    tangeran      banten  
1014    tangeran      banten  
1015    tangeran      banten  
1016    tangeran      banten  

[1017 rows x 8 columns]

In [ ]:
df.head()

Unnamed: 0                   place_name  \
0           0     Alas Purwo National Park   
1           1                    Green Bay   
2           2           De Djawatan Forest   
3           3              Banyuwangi Park   
4           4  Tempat wisata di banyuwangi   

                                        reviews_link  rating  total_reviews  \
0  https://serpapi.com/search.json?data_id=0x2dd3...     4.5           1449   
1  https://serpapi.com/search.json?data_id=0x2dd4...     4.7           2322   
2  https://serpapi.com/search.json?data_id=0x2dd3...     4.6           6906   
3  https://serpapi.com/search.json?data_id=0x2dd1...     4.5           1242   
4  https://serpapi.com/search.json?data_id=0x2dd1...     4.2             37   

                                             address        city    province  
0  Kutorejo, Kalipait, Tegaldlimo, Banyuwangi Reg...  banyuwangi  jawa timur  
1  Dusun Krajan, Sarongan, Pesanggaran, Banyuwang...  banyuwangi  jawa timur  
2  Purwosari, Benculuk, Kec. Cluring, Kabupaten B...  banyuwangi  jawa timur  
3  Dusun Krajan, Dadapan, Kec. Kabat, Kabupaten B...  banyuwangi  jawa timur  
4  perum adimas diponegoro No.02, Lingkungan Cuki...  banyuwangi  jawa timur

In [ ]:
df.tail()

Unnamed: 0                                  place_name  \
1012          15                          Telaga Biru Cisoka   
1013          16                     Danau Barat Alam Sutera   
1014          17                             Koja Cliff Park   
1015          18      Citra Raya World Of Wonders Theme Park   
1016          19  Kampung Gempol - Wisata Edukasi & Rekreasi   

                                           reviews_link  rating  \
1012  https://serpapi.com/search.json?data_id=0x2e42...     4.0   
1013  https://serpapi.com/search.json?data_id=0x2e69...     4.5   
1014  https://serpapi.com/search.json?data_id=0x2e42...     4.0   
1015  https://serpapi.com/search.json?data_id=0x2e42...     4.4   
1016  https://serpapi.com/search.json?data_id=0x2e69...     4.6   

      total_reviews                                            address  \
1012           6286  PCCJ+Q6X, Jalan Cigaru, Cisoka, Kec. Cisoka, K...   
1013            153  Jl. Kp. Kosong No.11, RT.001/RW.004, Panunggan...   
1014           1690  M9PX+55H, Cireundeu, Cikuya, Kec. Solear, Kabu...   
1015           4740  Citra Raya Cikupa Jl. Ecopolis Boulevard Utama...   
1016            215  Gg. H. Marjuki Jl. Kunciran Induk/Raya Gempol,...   

          city province  
1012  tangeran   banten  
1013  tangeran   banten  
1014  tangeran   banten  
1015  tangeran   banten  
1016  tangeran   banten

#**Business Understanding**

## **Business Objective**

Dataset Tourism Data on Java

Data berikut merupakan sebuah data pariwisata di pulau Jawa. Disini kita akan membantu dalam menganalisis agar destinasi wisata di pulau Jawa semakin berkembang.

Dalam dataset ini terdapat 1017 baris dan 8 kolom.

## **Assess Situation**

Dalam dataset ini, terdapat situasi yang dimana destinasi pariwisata yang mencakup di daerah pulau Jawa. Luasnya dataset ini membutuhkan peninjauan lebih lanjut agar perusahaan pariwisata dapat memahami lebih dalam mengenai lokasi wisata yang diminati.

## **Analitik Goals & Project Plan**

Dalam dataset ini akan dilakukan.


* Mengidentifikasi Total Rating dan Nilai Daya Tarik Lokasi
* Menghasilkan dan Menampilkan Visualisasi
* Menyediakan Dasar Pengambilan Keputusan


